config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -3356,8 +3357,12 @@ private ServerWebExchangeMatcher createAttemptAuthenticationRequestMatcher() {
 				return Collections.emptyMap();
 			}
 			Map<String, String> result = new HashMap<>();
-			registrations.iterator().forEachRemaining(
-					(r) -> result.put("/oauth2/authorization/" + r.getRegistrationId(), r.getClientName()));
+			registrations.iterator().forEachRemaining((r) -> {
+				if (AuthorizationGrantType.AUTHORIZATION_CODE.equals(r.getAuthorizationGrantType())
+						|| AuthorizationGrantType.IMPLICIT.equals(r.getAuthorizationGrantType())) {
```

Comment:
Let's remove `AuthorizationGrantType.IMPLICIT` since it's `@Deprecated`. Also, the only supported redirect-based flows is `AUTHORIZATION_CODE`.

Reply:
I removed the `IMPLICIT` case.